hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -2307,7 +2307,8 @@
             X'ref'ed with each IP test that requsts entropy from EDN.
             '''
       stage: V2
-      tests: ["chip_sw_edn_entropy_reqs"]
+      tests: ["chip_sw_edn_entropy_reqs",
+              "chip_sw_csrng_edn_concurrency"]
```

Comment:
@sha-ron i think this new test maybe we should add to the jitter list as well.

Reply:
@tjaychen Yes, we can add to the jitter list and create reduced frequency test.

Reply:
that would be great! thank you!